server/helpers/findEmail.js

```diff
@@ -0,0 +1,9 @@
+exports.findEmail = (allEmails, email) => {
+  let flag = false;
+  allEmails.forEach(element => {
```

Comment:
`Array.find` would be helpful here
Or you can simply use `Array.includes` directly in the controller w/o the need of this helper